tests/hazmat/primitives/test_padding.py

```diff
@@ -240,3 +247,96 @@ def test_bytearray(self):
         unpadder = padding.ANSIX923(128).unpadder()
         final = unpadder.update(padded) + unpadder.finalize()
         assert final == unpadded + unpadded
+
```

Comment:
I'm very ambivalent about the utility of these tests.

Part of the benefit of moving stuff to Rust is that we can have confidence that these are correct by construction. As is these are basically tests that pyo3's borrow checking code is correct.

Reply:
If you're comfortable with (eventually) shipping free-threaded wheels without explicit multithreaded tests for mutable primitives implemented in rest, I'm happy to delete the new tests.

Reply:
That's my strong inclination yes. (Ordinarily I'd bug @reaperhulk for a second opinion, but he just had a kid so I'm not going to nag him in too many reviews.)

Let me mull on this for a day or two before we make any decisions.

Reply:
Yeah, after some review, I don't think we should carry tests that are really just tests of pyo3's invariants.